### ISSUE TITLE
Minor bug while generating route

### ISSUE THREAD

**Issue by KarandikarMihir** [mxstbr/react-boilerplate#809]
Hi there,
I found a minor bug while creating a new route with `npm run generate`.

Bug: Sagas are loaded in `routes.js` asynchronously even if I have chosen NOT to create a sagas.js file using `npm run generate`. 

Steps for reproduction:
1. `npm run generate`
2. select "container"
3. Styling: NO, actions/const/reducer/selectors: YES, sagas: NO.
4. `npm run generate`
5. select "route"
6. Provide container name and pathname. 

After following these steps, you can see this in `createRoutes()` inside `routes.js` file:

```
   {
      path: 'testform',
      name: 'testForm',
      getComponent(nextState, cb) {
        const importModules = Promise.all([
          System.import('containers/TestForm/reducer'),
          System.import('containers/TestForm/sagas'), //sagas are non-existent
          System.import('containers/TestForm'),
        ]);

        const renderRoute = loadModule(cb);

        importModules.then(([reducer, sagas, component]) => { //sagas are non-existent
          injectReducer('testForm', reducer.default);
          injectSagas(sagas.default); // sagas are non-existent
          renderRoute(component);
        });

        importModules.catch(errorLoading);
      },
    }, 
```

Is this a bug? Or am I missing something? Ideally, we should be able to see the container in browser right after creating it container and its route. But an error is thrown saying that "sagas.js" cannot be found.

**Comment by gihrig** [comment]
@KarandikarMihir The issue you are reporting sounds like the one fixed in #762.

Are you testing against the latest commit?

**Comment by KarandikarMihir** [comment]
@gihrig Oh. I was running an older boilerplate. Closing.
